viewer/README.md

```diff
@@ -1,11 +1,12 @@
 # Vision Zero Viewer (VZV)
 
-Production site: https://visionzero.austin.gov/viewer/
-Staging site: https://visionzero-staging.austinmobility.io/viewer/
-
+![Vision Zero Viewer](../docs/images/vzv.png)
```

Comment:
I don't this image loading in the rich diff of this file but I can't see anything wrong with the path. I see the file in the repo too 🤔

Reply:
thanks, Mike!!